hbase-it/src/test/java/org/apache/hadoop/hbase/RESTApiClusterManager.java

```diff
@@ -513,11 +513,7 @@ private boolean hasCommandCompletedSuccessfully(final long commandId) {
           throw new RuntimeException("retries exhausted", e);
         }
       }
-      try {
-        retryCounter.sleepUntilNextRetry();
-      } catch (InterruptedException e) {
-        throw new RuntimeException(e);
-      }
+      retryCounter.sleepUntilNextRetry();
```

Comment:
Here also we are changing the behave. Previously throw RTE when interrupted. But now this is been changed  Main thing is even if interrupted, the RetryCounter will make sure the thread been slept for the specified time (Which might not be really wanted some times)

Reply:
I believe the purpose of retryCounter.sleepUntilNextRetry() should be uninterrupted sleep because RetryCounter is mainly being used by retries with sleeps and retries with different backoff policies. In such scenario, RetryCounter being a library should not ideally throw InterruptedException even if sleep is interrupted because it is being retried by clients to achieve certain tasks.